_scripts/fetch-library-samples.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# Get the Library docs
+svn co https://github.com/docker-library/docs/trunk ./_samples/library || (echo "Failed library download" && exit 1)
```

Comment:
@thaJeztah sorry this hasn't been communicated clearly in a previous PR. We have updated the library logic to have the sample library links point to Hub pages instead. I don't think fetching them is still necessary except for archived versions. Let me double check on this though.

Reply:
Yeah, I never liked the samples being there, but it was still in the dockerfile, so I kept it

Reply:
oh, looks like the "samples home" https://docs.docker.com/samples/ links to the docker hub pages, but the navigation still has the "fetched" versions as well

<img width="291" alt="screenshot 2019-03-07 at 23 26 35" src="https://user-images.githubusercontent.com/1804568/53993742-7fceb680-4130-11e9-9665-0c907bbab8fb.png">

And those pages are still active; https://docs.docker.com/samples/library/alpine/

I can work on turning those into redirects, and remove them from the actual docs (that may need a bit of thinking, so I'd prefer doing that in a follow-up to not uphold the fixes in this PR)

Reply:
Both of those comments are accurate - just haven't had the time to come back and clean those up. :( Thank you - will merge this one then!